resources/public/api/conf/1.0/examples/ChargesSummary.get.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
I thought we agreed not to use Json for test data but objects/vals

Reply:
👍